Framework/DataHandling/src/LoadMask.cpp

```diff
@@ -120,47 +345,35 @@ void LoadMask::exec() {
   } else if (boost::ends_with(filename, "k") ||
              boost::ends_with(filename, "K")) {
     // 2.2 ISIS Masking file
-    loadISISMaskFile(filename, m_MaskSpecID);
+    loadISISMaskFile(filename, m_maskSpecID);
     m_defaultToUse = true;
   } else {
     g_log.error() << "File " << filename << " is not in supported format. \n";
     return;
   }
-
   // 3. Translate and set geometry
   g_log.information() << "To Mask: \n";
-  std::vector<int32_t> maskdetids;
-  std::vector<int32_t> maskdetidpairsL;
-  std::vector<int32_t> maskdetidpairsU;
-
-  componentToDetectors(mask_bankid_single, maskdetids);
-  detectorToDetectors(mask_detid_single, mask_detid_pair_low,
-                      mask_detid_pair_up, maskdetids, maskdetidpairsL,
-                      maskdetidpairsU);
-
-  g_log.information() << "To UnMask: \n";
-  std::vector<int32_t> unmaskdetids;
-  std::vector<int32_t> unmaskdetidpairsL;
-  std::vector<int32_t> unmaskdetidpairsU;
-
-  this->bankToDetectors(unmask_bankid_single, unmaskdetids, unmaskdetidpairsL,
-                        unmaskdetidpairsU);
-  this->detectorToDetectors(unmask_detid_single, unmask_detid_pair_low,
-                            unmask_detid_pair_up, unmaskdetids,
-                            unmaskdetidpairsL, unmaskdetidpairsU);
+
+  this->componentToDetectors(m_maskCompIdSingle, m_maskDetID);
+
+  // unmasking is not implemented
+  // g_log.information() << "To UnMask: \n";
```

Comment:
Masking is often used and it would be pretty annoying to see "Unmasking not implemented" for every masking operation. The warning should be issued if user tries to run unmasking, so this should happen in hml, when unmasking is requested. But there are no current key to request unmasking, and introducing it would go far behind the scope of this ticket.